.github/workflows/templates-api.yml

```diff
@@ -24,28 +25,70 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Create artifacts directory
+        run: mkdir -p /tmp/artifacts
+
+      - name: Install gron tool
+        run: |
+          url="https://github.com/tomnomnom/gron/releases/download/v0.7.1/gron-linux-amd64-0.7.1.tgz"
+          curl -L "$url" | tar -xz -C /usr/local/bin
+
```

Comment:
V k8s definiciach sa jednoducho spravi preklep, ... nieco testujeme v testoch, ale napr. ci je tam persistent alebo volatile storage, to sa testuje tazko.

Preto som v tomto commite pridal do CI, aby vygenerovalo k8s diff a poslal ho ako comment do PR.
Tym by sa mohlo zabranit neumyselnym zmenam (napr. pri update etcd helm chart-u).

-----

Standardne to riesi `kubectl diff` alebo `helm diff`, no kedze to tu v tomto projekte mixujeme spolu, tak sa mi nepodarilo pomocou toho dosiahnut poriadny vysledok.

-----

Ak by sme to chceli mat vsetko cez `helm`, tak to by bolo idealne, ... no mala by existovat nejaka `common` helm kniznica, ktora nas oddeli od rozdielov medzi AWS/Azure stackom. To co je teraz v `cloud` adresary a `aws/azure.sh`.

Do toho som sa nechcel pustat.

-----

Potom som skusal https://github.com/homeport/dyff, ktory ma aj nejaku podporu pre k8s.
To bolo uz lepsie, ale nevedelo to detekovat random `<hash>`, ktorych na niektorych miestach pridava k8s (deployment, replica set, ...) k podu.

----

Su na to aj nejake dalsie utilitky, ale nic poriadne.
Tak som to vyriesil tak, ze sa pomocou `kubect get ...` dumpne old/new k8s state ako `json`.

Pomocou `gron` sa prevedie na `key = value` pairs, ktore sa daju diffovat.
![image](https://user-images.githubusercontent.com/19371734/200930313-878dbb35-889e-462c-8ac1-7640cc9c32e9.png)